Cargo.toml

```diff
@@ -1,27 +1,27 @@
 [package]
 name = "cosmic-ext-applet-system-monitor"
-version = "0.1.4"
-edition = "2021"
+version = "0.1.5"
+edition = "2024"
 authors = ["D-Brox <davidfbg00@gmail.com>"]
 description = "System monitor applet for the COSMIC DE"
 readme = "README.md"
 license = "GPL-3.0"
 homepage = "https://github.com/D-Brox/cosmic-ext-applet-system-monitor"
 repository = "https://github.com/D-Brox/cosmic-ext-applet-system-monitor.git"
-rust-version = "1.80"
+rust-version = "1.85"
 
 [dependencies]
 circular-queue = "0.2.7"
 i18n-embed-fl = "0.9.1"
-# lazy-regex = "3.4.1"
-# nvml-wrapper = "0.10.0"
+lazy-regex = "3.4.1"
+nvml-wrapper = "0.10.0"
```

Comment:
I'd recommend pulling from git because it has a lot of useful fixes and features. You can set the rev to `572095f63` to lock it at the current commit.

```toml
[dependencies.nvml-wrapper]
git = "https://github.com/rust-nvml/nvml-wrapper"
rev = "572095f6"
```

This also adds fan control and overclocking features.

Reply:
Those features aren't really needed for the purposes of this system monitor, but I'll check if there is anything that could affect it

Reply:
Ok, nvm, they added Blackwell support a bit after the 0.10.0 release, I'll pull from git